org.octave.Octave.json

```diff
@@ -0,0 +1,394 @@
+      "config-opts": [
+        "--enable-shared=yes"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/1.3.26/GraphicsMagick-1.3.26.tar.bz2",
+          "sha256": "054bd6e10c6d82b6adb36d1a9f860e9db90689d3b61e77a344d2f962b33167e5"
+        }
+      ]
+    },
+    {
+      "name": "arpack-ng",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/opencollab/arpack-ng/archive/3.5.0.tar.gz",
+          "sha256": "50f7a3e3aec2e08e732a487919262238f8504c3ef927246ec3495617dde81239"
+        }
+      ]
+    },
+    {
+      "name": "fftw3",
+      "no-autogen": true,
+      "config-opts": [
+        "--enable-shared",
+		"--enable-threads"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.fftw.org/fftw-3.3.6-pl2.tar.gz",
+          "sha256": "a5de35c5c824a78a058ca54278c706cdf3d4abba1c56b63531c2cb05f5d57da2"
+        }
+      ]
+    },
+    {
+      "name": "fftw3f",
+      "no-autogen": true,
+      "config-opts": [
+        "--enable-shared",
+        "--enable-threads",
+        "--enable-float"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.fftw.org/fftw-3.3.6-pl2.tar.gz",
+          "sha256": "a5de35c5c824a78a058ca54278c706cdf3d4abba1c56b63531c2cb05f5d57da2"
+        }
+      ]
+    },
+    {
+      "name": "mesa",
+      "config-opts": [
+        "--enable-osmesa"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://mesa.freedesktop.org/archive/mesa-17.2.0-rc6.tar.gz",
+          "sha256": "ce25345b7da886cdc730299065e2f27ac285eddc75c89595be16cb8ec8497fcc"
+        }
+      ]
+    },
+    {
+      "name": "sundials-ida",
+      "buildsystem": "cmake",
+      "config-opts": [
+        "-DKLU_ENABLE=ON",
+        "-DKLU_INCLUDE_DIR:PATH=/app/include",
+        "-DKLU_LIBRARY_DIR:PATH=/app/lib"
+      ],
+      "builddir": true,
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://computation.llnl.gov/projects/sundials/download/ida-2.9.0.tar.gz",
+          "sha256": "37dee0613c00beb35c1fa08f4ba0786cb97e5e6a834612d68b1e41e9282fd1de"
+        }
+      ]
+    },
+    {
+      "name": "qscintilla",
+      "no-autogen": true,
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/pyqt/files/QScintilla2/QScintilla-2.10/QScintilla_gpl-2.10.tar.gz",
+          "sha256": "16be30577bc178470936c458551f2512cc068aff6e7a7de6ed244e28c045f6ec"
+        },
+        {
+          "type": "patch",
+          "path": "qscintilla.patch"
+        },
+        {
+			"type": "shell",
+			"commands": [
+				"cd Qt4Qt5 && qmake && make install"
+			]
+        },
+		{
+			"type": "shell",
+			"commands": [
+				"echo 'all:\n\ninstall:\n\techo \'Already done installing\'' > Makefile"
```

Comment:
This is a workaround to allow Octave being build with Flatpak stable, which does not yet support simple build systems.

Reply:
The current release is stable you mean the LTS version which is not relevant to flathub at all and I would prefer readable manifests over the passing by Debian user.

Reply:
Okay, done.